### DISCUSSION TITLE
Duplicate Customer Note Emails

### DISCUSSION THREAD

**Discussion by unitv** [woocommerce/woocommerce#39330] [General]
My customers are receiving duplicate emails when I apply Customer Note
I'm using mercadopago gateway.
How to fix it?
Thanks.

**Comment by barryhughes** [comment]
Hi! If you are using the official [Mercado Pago Checkout](https://woocommerce.com/products/mercado-pago-checkout/) and only see this bug when that is in use (ie, you've tested with Direct Bank Payments or similar and it doesn't happen there), I'd suggest reaching out to the [Mercado Pago](https://woocommerce.com/vendor/mercado-pago/) team for further support :+1:
